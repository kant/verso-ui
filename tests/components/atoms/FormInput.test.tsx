import React from 'react'
import { jsx } from '@emotion/react'
import renderer from 'react-test-renderer'
import { TextInput, TextArea } from '../../../src'

test('Snapshot Test: TextInput', () => {
  const component = renderer.create(
    <TextInput
      type="text"
      label="Text input label"
    />,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

test('Snapshot Test: TextArea', () => {
  const component = renderer.create(
    <TextArea
      label="Secondary Button"
    />,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
