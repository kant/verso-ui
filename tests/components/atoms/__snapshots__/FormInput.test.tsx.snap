// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test: TextArea 1`] = `
<div
  className="css-z21lfh-TextArea"
>
  <label
    className="css-1bawu2g-TextArea"
    htmlFor="Secondary Button"
  >
    Secondary Button
  </label>
  <textarea
    className="css-1jf3zmb-TextArea"
    id="Secondary Button"
    name="Secondary Button"
  />
</div>
`;

exports[`Snapshot Test: TextInput 1`] = `
<div
  className="css-6r466p-TextInput"
>
  <label
    className="css-2x82w0-TextInput"
    htmlFor="Text input label"
  >
    Text input label
  </label>
  <input
    className="css-18ek46i-TextInput"
    id="Text input label"
    name="Text input label"
    type="text"
  />
</div>
`;
